README.md

```diff
@@ -1,2 +1,98 @@
 # next-common-files
+
+[travis-url]:https://travis-ci.org/moxystudio/next-common-files
+[travis-image]:http://img.shields.io/travis/moxystudio/next-common-files/master.svg
+[codecov-url]:https://codecov.io/gh/moxystudio/next-common-files
+[codecov-image]:https://img.shields.io/codecov/c/github/moxystudio/next-common-files/master.svg
+
 Next.js plugins that insert loaders for common files.
+
+Projects developed with Next.js have to manually insert rules in the configuration file for handling file types that are potentially very common across multiple projects.
+
+These plugins quicken the initial setup of a project by removing that effort from the process, demanding less time from the developer.
+
+## Installation
+``` npm install --save-dev @moxy/next-common-files ```
+
+## Usage
+```js
+// next.config.js
+const { withRasterImages, withPlayback, withSVG } = require('@moxy/next-common-plugins);
+
+module.exports = withRasterImages({
+	/* options */
+});
+```
+You can use `next-compose-plugins` to easily call multiple plugins.
+```js
+// next.config.js
+const withPlugins = require('next-compose-plugins');
+
+module.exports = (phase, nextConfig) => {
+	withPlugins([
+		[withRasterImages()],
+		[withPlayback()],
+	]),
+};
+```
+
+## Loaders
```

Comment:
I have a suggestion for the whole readme. The way I see it is:

1. Change `Usage` to have the typical usage, that is, have all 3 plugins enabled.
2. Add a `API` section, with all the documentation segmented per plugin, including the options available for each one.

Reply:
I'm adding the `API` section, but i'm not sure how much information i should transfer to it. With the exception of the `svg` loader, which can toggle to a different set of loaders than the default, every plugin uses the same configuration and it would become very repetitive to explain the same usage details on each. For example, the `svg` loader has more details to explore, but with the exception of file types, `raster-images` and `playback` would have identical sections.

Also, because most options you can apply are from other packages (e.g., the rule options like `include` and `exclude` are from `webpack`, and `limit` and `fallback` are from `url-loader`), i'm unsure whether to explore those in my own `README` or direct the reader towards their official documentation. So far, i'm doing a bit of both, but i'm not sure if separating it like this might be unnecessary, and require undesired upkeep over time to keep up with changes in those packages.